Source/Private/ParseBicep.ps1

```diff
@@ -1,3 +1,17 @@
+function WriteBicepDiagnostic {
```

Comment:
Should `WriteBicepDiagnostics` move to a `WriteBicepDiagnostics.ps1` file maybe? I guess that we will re-use it for `ConvertTo-Bicep`

Reply:
Definitely a good idea! I'll take a look at separating that.